starter-code/public/scripts/models/repo.js

```diff
@@ -7,11 +7,19 @@ var app = app || {};
   repos.all = [];
 
   repos.requestRepos = function(callback) {
-    // TODO: How would you like to fetch your repos? Don't forget to call the callback.
+    // DONE: How would you like to fetch your repos? Don't forget to call the callback.
     //       Remember that the callback function we'll want to call relies on repos.all
     //       being an array with a bunch of repo objects in it, so you'll need to
     //       populate it with the response from Github before you call the callback.
-
+    $.ajax({
+      url: 'https://api.github.com/users/sharmarkei/repos',
+      method: 'GET',
+      headers: {'Authorization': 'token ' + githubToken}
+      .then(function(data){
```

Comment:
`.then(function(data) {` should not be within `$.ajax` call. 
It should be moved outside.